app/assets/scss/application.scss

```diff
@@ -6,3 +6,13 @@
 body {
   @extend .govuk-body;
 }
+
+.panel-background {
```

Comment:
I know panel in invision has a different color but I would go for the color palette in the GDS as in the rest of the app. I think invision design in this case is wrong. We can confirm this with UX.